modules/tile-converter/src/deps-installer/deps-installer.ts

```diff
@@ -47,7 +47,7 @@ export class DepsInstaller {
     const childProcess = new ChildProcessProxy();
     await childProcess.start({
       command: 'npm',
-      arguments: ['install', 'sharp', 'join-images'],
+      arguments: ['install', 'sharp@^0.30.4', 'join-images@^1.1.3'],
```

Comment:
These things tend to become difficult to maintain unless we add a comment explaining why we pinned these deps. Later when someone needs to bump them, it must be clear what to look out for.

Reply:
Done